src/components/select/select-list/select-list.component.js

```diff
@@ -461,7 +462,6 @@ const SelectList = React.forwardRef(
           >
             <StyledSelectListContainer
               data-element="select-list-wrapper"
-              height={listHeight}
```

Comment:
since we're removing this (which I agree is not needed any more), we should also remove all references to `listHeight` and `setListHeight` elsewhere in the component, as they're not used anymore. That means the whole `useLayoutEffect `that sets the height can also be removed!

Reply:
It's actually still used to set height on `StyledPopoverContainer`

Reply:
oh yes, sorry - not sure how I missed that